src/churn/churn.ts

```diff
@@ -172,8 +172,14 @@ var log :logging.Log = new logging.Log('churn');
   };
 
   // Generates a key suitable for use with CaesarCipher, viz. 1-255.
-  var generateCaesarKey_ = () : number => {
-    return (random.randomUint32() % 254) + 1;
+  var generateCaesarKey_ = (): number => {
+    try {
+      return (random.randomUint32() % 254) + 1;
```

Comment:
but we want a number from 1-255, so generate a number between 0-254, then add 1

(right?)